commands/user.bee.inc

```diff
@@ -98,6 +187,180 @@ function users_bee_callback($arguments, $options) {
+  $batch['progressive'] = FALSE;
+  batch_process();
+
+  bee_message(bt("User '!user' has been removed.", array(
+    '!user' => $user->name,
+  )), 'success');
+}
+
+/**
+ * Command callback: Block a given user.
+ */
+function user_block_bee_callback($arguments, $options) {
+  // Load the user account.
+  $user = user_load_by_name($arguments['username']);
+  if (!$user) {
+    bee_message(bt("User account '!user' cound not be found.", array(
+      '!user' => $arguments['username'],
+    )), 'error');
+    return;
+  }
+
+  $user->status = 0;
+  user_save($user);
+
+  bee_message(bt("User '!user' has been blocked.", array(
+    '!user' => $user->name,
+  )), 'success');
+}
+
+/**
+ * Command callback: Unblock a given user.
+ */
+function user_unblock_bee_callback($arguments, $options) {
+  // Load the user account.
+  $user = user_load_by_name($arguments['username']);
+  if (!$user) {
+    bee_message(bt("User account '!user' cound not be found.", array(
+      '!user' => $arguments['username'],
+    )), 'error');
+    return;
+  }
+
+  $user->status = 1;
+  user_save($user);
+
+  bee_message(bt("User '!user' has been unblocked.", array(
+    '!user' => $user->name,
+  )), 'success');
+}
+
+/**
+ * Command callback: Add a role to a given user.
+ */
+function user_add_role_bee_callback($arguments, $options) {
+  $user = user_load_by_name($arguments['username']);
+  if (!$user) {
+    bee_message(bt("User account '!user' cound not be found.", array(
+      '!user' => $arguments['username'],
+    )), 'error');
+    return;
+  }
+  $role = new stdClass();
+  $role->name = $arguments['role'];
+  $role->label = $arguments['role'];
+  $roles = user_roles(TRUE);
+  if(user_role_save($role)) {
+    if(!user_has_role($arguments['role'])){
+      if (!in_array($arguments['role'], $user->roles)) {
+        $user->roles[] = $arguments['role'];
+        user_save($user);
+        bee_message(bt("The role !role has been assigned to user '!user'.", array(
+          '!user' => $user->name, '!role' => $arguments['role']
+        )), 'success');
+      }else{
+            bee_message(bt("The role !role has allready been assigned to user '!user'.", array(
+              '!user' => $user->name, '!role' => $arguments['role']
+            )));
+      }
+    }
+  }
+}
+/**
+ * Command callback: Remove a role from a given user.
+ */
+function user_remove_role_bee_callback($arguments, $options) {
+  $user = user_load_by_name($arguments['username']);
+  if (!$user) {
+    bee_message(bt("User account '!user' cound not be found.", array(
+      '!user' => $arguments['username'],
+    )), 'error');
+    return;
+  }
+  if (in_array($arguments['role'], $user->roles)) {
+    if (($key = array_search($arguments['role'], $user->roles)) !== FALSE) {
+      unset($user->roles[$key]);
+    }
+    user_save($user);
+      bee_message(bt("The role !role has been removed from  user '!user'.", array(
+        '!user' => $user->name, '!role' => $arguments['role']
```

Comment:
It is preferable, once we've split lines for the array from the opening `array(` to put each array item on a separate line.
```suggestion
        '!user' => $user->name,
        '!role' => $arguments['role'],
```